app/mailer/index.js

```diff
@@ -48,8 +32,24 @@ function sendAsAdministrator(options) {
   return send(augmentedOptions);
 }
 
+function send(options) {
+  return apiInstance.sendTransacEmail({
+    sender: {
+      email: options.from.email,
+      name: options.from.name
+    },
+    to: [{
+      email: options.to.email,
+      name: options.to.name || ''
```

Comment:
```suggestion
      name: options.to.name
```

Empty `name` is not supported and would fail in any case.